stdlib/experimental/experimental.flux

```diff
@@ -245,7 +245,7 @@ builtin set : (<-tables: stream[A], o: B) => stream[C] where A: Record, B: Recor
 
 // to writes _pivoted_ data to an InfluxDB 2.x or InfluxDB Cloud bucket.
 //
-// **Deprecated**: `experimental.to()` is deprecated in favor of [`wideTo()`](https://docs.influxdata.com/flux/v0.x/stdlib/influxdata/influxdb/wideTo/),
```

Comment:
Other docs links use the full URL, is there a reason we want to change this one? The change to lower case makes sense.

Reply:
we use the shortened URL in case these links are being tested on a staging environment